docs/hw-platforms.rst

```diff
@@ -0,0 +1,182 @@
+Supported Boards
+=================
+
+The following Infineon products are supported by this library:
+
+KIT_XMC14_2GO
+------------------------------
+
+.. image:: img/KIT_XMC14_2GO.png
+    :width: 500
+
+The XMC1400 2Go kit provides an easy way to evaluate almost all capabilities of the XMC1400 microcontroller.  
+The kit is powered via USB, interfaces  to other Infineon sensor boards and provides multiple interfaces including a 
+CAN bus. The software development is supported via ModusToolbox™ and the Arduino IDE.  
+To find out more visit the Infineon homepage of the `KIT_XMC14_2GO`_.
+
+Pinout Diagram
+^^^^^^^^^^^^^^
+
+.. image:: img/XMC_4700_RelaxKit_5VShields.png
+    :width: 600
+
+
+
+
+
+
+KIT_XMC11_BOOT_001
+-------------------
```

Comment:
The naming is a bit weird for me.
The product name doesn't match the name inside the pinout diagram/IDE (code)

Reply:
Yep, the SCREAMING_SNAKE_CASE for the product ordering number also does not feel the best for me. 
![image](https://github.com/user-attachments/assets/4523d67c-b339-46e5-aa7f-596e2daf6433)
I would add not the product page title. The second one ?